data-small/allerton-bywater/mode-split.csv

```diff
@@ -1,8 +1,8 @@
 distance_band,proportion_in_distance_band,all,trimode,walk_base,cycle_base,drive_base,other_base,walk_goactive,cycle_goactive,drive_goactive,other_goactive,percent_walk_base,percent_cycle_base,percent_drive_base,percent_other_base,percent_walk_goactive,percent_cycle_goactive,percent_drive_goactive,percent_other_goactive
-0-3,11,58,47,13,1,33,11,20,19,8,11,22,2,57,19,34,33,14,19
-3-6,12,68,50,3,1,47,18,1,10,40,18,4,1,69,26,1,15,59,26
-6-10,16,86,72,1,1,70,14,1,6,65,14,1,1,81,16,1,7,76,16
-10-15,37,203,137,2,2,133,65,1,4,132,65,1,1,66,32,0,2,65,32
-15-20,10,55,46,1,0,46,9,1,0,46,9,2,0,84,16,2,0,84,16
+0-3,12,66,55,16,2,37,11,18,17,20,11,24,3,56,17,27,26,30,17
```

Comment:
Looking good, great to see those extra trips in there.

Reply:
Quick question for you @natesheehan, what is causing the changes here?

Reply:
`distance_band,proportion_in_distance_band,all,trimode,walk_base,cycle_base,drive_base,other_base,walk_goactive,cycle_goactive,drive_goactive,other_goactive,percent_walk_base,percent_cycle_base,percent_drive_base,percent_other_base,percent_walk_goactive,percent_cycle_goactive,percent_drive_goactive,percent_other_goactive`

If any of the above were calculated using the desire_lines_few file then this is where the change would have occurred. The affected sites, all had invalid data structures for the town element of the OD dataframe so, I believe this is where the addition of trips is now coming in.

Reply:
Makes sense and what I was thinking, only thing was some were going up and some down. Great fix in any case!

Reply:
Thanks! But yes that sure is strange. First thoughts: the distance of the town affects the distribution of the total distance band? I can try rerun with a site that was not affected and see if there are any variances in mode split  - as if so, I am assuming it's a deterministic issue?

Reply:
>  I can try rerun with a site that was not affected and see if there are any variances in mode split - as if so, I am assuming it's a deterministic issue?

That's a good plan. It could be because the OSM network or routing services have changed but the main thing is total counts. If you do a basic sanity check to see things add-up, unlike in #100, that would be greatly appreciated.